libs/flowbite-angular/src/lib/components/navbar/navbar-icon-button.component.ts

```diff
@@ -0,0 +1,32 @@
+import type { DeepPartial } from '../../common';
+import { createClass } from '../../utils';
+import { BaseComponent } from '../base-component.directive';
+import type { NavbarIconButtonClass, NavbarIconButtonTheme } from './navbar-icon-button.theme';
+import { NavbarIconButtonThemeService } from './navbar-icon-button.theme.service';
+
+import { Component, inject, input, signal } from '@angular/core';
+
+@Component({
+  selector: 'flowbite-navbar-icon-button',
+  standalone: true,
+  imports: [],
+  template: `<ng-content />`,
+})
+export class NavbarIconButtonComponent extends BaseComponent {
+  public readonly themeService = inject(NavbarIconButtonThemeService);
+
```

Comment:
Inject navbarcomponent and navbarcontent component
And in the verify function (inheritage from base directive), check if either the navbar component or navbarcontent component is not undefined, otherwise, throw a new error (look at other component you will have a text for the error)

Reply:
Double check I've done that right in my recent commit :)

Reply:
Yupp perfect